website/content/en/pre-docs/getting-started/_index.md

```diff
@@ -294,7 +294,7 @@ kubectl delete node $NODE_NAME
 
 ## Cleanup
 
-To avoid additional charges, remove the demo infrastructure from your AWS account.
+To avoid additional charges, remove the demo infrastructure from your AWS account.  Party.
```

Comment:
?

Reply:
I think I typed this because I couldn't figure out why Hugo wasn't updating. I was on karpenter.sh and not localhost............